app/views/roles/index.html.erb

```diff
@@ -35,4 +35,4 @@
     <% end %>
   </tbody>
 </table>
-<%= will_paginate_with_info @roles %>
+<%= will_paginate_with_info @roles %>
```

Comment:
Nit: please keep the newline at the end

Reply:
> How do the various lengths work in translated variants?
Good point, there were translations longer than my hardcoded width value. I removed the fixed width for columns, now the change is just displaying full role name. Also added new line at the end.